s3torchconnectorclient/python/tst/integration/test_mountpoint_s3_integration.py

```diff
@@ -3,22 +3,27 @@
 import hashlib
 import logging
 import math
+import os
 import pickle
 import sys
 import uuid
 import random
 import pytest
 
+os.environ["S3_CONNECTOR_ENABLE_CRT_LOGS"] = "DEBUG"
+
 from s3torchconnectorclient._mountpoint_s3_client import (
     MountpointS3Client,
     S3Exception,
     ListObjectStream,
 )
 
+from s3torchconnectorclient import LOG_TRACE
+
 logging.basicConfig(
     format="%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s"
 )
-logging.getLogger().setLevel(1)
+logging.getLogger().setLevel(LOG_TRACE)
```

Comment:
Given we only had `LOG_TRACE` because we were exposing Rust logs to Python, we can probably remove it